README.md

```diff
@@ -21,34 +21,80 @@ Whether you’re an experienced developer, a PM or designer, Bolt.new allows you
 
 For developers interested in building their own AI-powered development tools with WebContainers, check out the open-source Bolt codebase in this repo!
 
-## Tips and Tricks
+## Prerequisites
 
-Here are some tips to get the most out of Bolt.new:
+Before you begin, ensure you have the following installed:
 
-- **Be specific about your stack**: If you want to use specific frameworks or libraries (like Astro, Tailwind, ShadCN, or any other popular JavaScript framework), mention them in your initial prompt to ensure Bolt scaffolds the project accordingly.
+- Node.js (v20.15.1)
+- pnpm (v9.4.0)
 
-- **Use the enhance prompt icon**: Before sending your prompt, try clicking the 'enhance' icon to have the AI model help you refine your prompt, then edit the results before submitting.
+## Setup
 
-- **Scaffold the basics first, then add features**: Make sure the basic structure of your application is in place before diving into more advanced functionality. This helps Bolt understand the foundation of your project and ensure everything is wired up right before building out more advanced functionality.
+1. Clone the repository (if you haven't already):
 
-- **Batch simple instructions**: Save time by combining simple instructions into one message. For example, you can ask Bolt to change the color scheme, add mobile responsiveness, and restart the dev server, all in one go saving you time and reducing API credit consumption significantly.
+```bash
+git clone https://github.com/coleam00/bolt.new-any-llm.git
```

Comment:
repository name is pointing to coleam00's github

Reply:
I responded with  [e141171](https://github.com/stackblitz/bolt.new/pull/733/commits/e14117113250213e2ed3db939900fd4d211384af)